### ISSUE TITLE
32 Bit JIT Builder Travis Builds fail

### ISSUE THREAD

**Issue by mgaudet** [eclipse/omr#703]
In #702 We are attempting to explicitly set PLATFORM to ensure we produce and test a 32 bit JIT when SPEC is set to a 32 bit linux configuration. 

However, [the Travis builds continues to fail in this scenario](https://travis-ci.org/eclipse/omr/builds/195009170) because of a mismatch between the architecture of the JITBuilder archive. 

```
-Wl,-z,origin,-rpath,\$ORIGIN,--disable-new-dtags,-rpath-link,../..
make[2]: Leaving directory `/home/travis/build/eclipse/omr/fvtest/vmtest'
make -C jitbuilder/release all
make[2]: Entering directory `/home/travis/build/eclipse/omr/jitbuilder/release'
g++ -o Call.o -g -std=c++0x -O2 -c -fno-rtti -fPIC -I./include/compiler -I./include src/Call.cpp
g++ -g -fno-rtti -o call Call.o -L. -ljitbuilder -ldl
/usr/bin/ld: skipping incompatible ./libjitbuilder.a when searching for -ljitbuilder
/usr/bin/ld: cannot find -ljitbuilder
collect2: ld returned 1 exit status
make[2]: *** [call] Error 1
make[2]: Leaving directory `/home/travis/build/eclipse/omr/jitbuilder/release'
make[1]: *** [jitbuilder/release] Error 2
make[1]: Leaving directory `/home/travis/build/eclipse/omr'
make: *** [tests] Error 2
```

**Comment by mgaudet** [comment]
Also, it turns out in internal testing, 32 Bit JitBuilder tests were also never hooked up.

**Comment by mgaudet** [comment]
Closing this -- Fixed by #707
